python/pyspark/sql/connect/proto/base_pb2.py

```diff
@@ -16,13 +16,19 @@
 #
 # -*- coding: utf-8 -*-
 # Generated by the protocol buffer compiler.  DO NOT EDIT!
+# NO CHECKED-IN PROTOBUF GENCODE
 # source: spark/connect/base.proto
+# Protobuf Python Version: 5.28.3
 """Generated protocol buffer code."""
-from google.protobuf.internal import builder as _builder
 from google.protobuf import descriptor as _descriptor
 from google.protobuf import descriptor_pool as _descriptor_pool
+from google.protobuf import runtime_version as _runtime_version
 from google.protobuf import symbol_database as _symbol_database
+from google.protobuf.internal import builder as _builder
 
+_runtime_version.ValidateProtobufRuntimeVersion(
+    _runtime_version.Domain.PUBLIC, 5, 28, 3, "", "spark/connect/base.proto"
```

Comment:
Actually let's don't do this for now. This requires the specific protobuf version strictly, and it loses the compatibility. For example, I and all people (who I know) cannot run tests in Mac because of the test failures by `protobuf` 5.28.3 being incompatible in Mac for some reasons.